components/DepartmentsWalkthrough.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+  <v-container class="Category-Background carousel-view" fluid>
+    <transition-group class="carousel" tag="div">
+      <div
+        v-for="(department, i) in departments"
+        v-bind:key="department.department"
+        v-bind:style="{display: department.display}"
+        class="slide"
+      >
+        <v-row>
+          <v-col class="Category-Title">{{i+1}}. {{department.header}}</v-col>
+        </v-row>
+        <v-row class="Category-Content">
+          <v-col cols="7">
+            <v-row class="Category-Content-Text text-sm-body-1">
+              <span v-html="department.content"></span>
+            </v-row>
+            <v-row justify="center">
+              <v-btn id="btn" elevation="0" outlined color="#BDBDBD" @click="expand">SEE MORE v</v-btn>
+            </v-row>
+          </v-col>
+          <v-col cols="1"></v-col>
+          <v-col>
+            <v-row>
+              <div class="Category-Slider">
+                <div class="Slider-Amount">${{department.storedValue}} mil</div>
+                <v-row justify="center">
+                  <v-slider
+                    v-bind:max="department.sliderMax"
+                    v-model="department.storedValue"
+                    min="0"
+                    vertical
+                    track-color="#B6DADA"
+                    color="#2A6465"
+                    @mousedown="tradeBtns"
+                  />
+                </v-row>
+              </div>
+            </v-row>
+            <v-row
+              class="Category-Budget-Caption text-sm-body-1"
+            >Last year’s budget: {{department.budget}}</v-row>
+          </v-col>
+        </v-row>
+      </div>
+    </transition-group>
+    <div class="carousel-controls">
+      <v-btn
+        v-for="(button) in buttonsShow"
+        v-bind:key="button.name"
+        class="carousel-controls__button"
+        v-bind:class="button.class"
+        v-bind:disabled="button.disabled"
+        v-bind:outlined="button.outlined"
+        rounded
+        depressed
+        width="240px"
```

Comment:
can you style width / color with the class `carousel-controls__button` instead?
(honest question -- it's ok if the answer is no b/c of how vuetify works)

Reply:
i tried and unfortunately not!